packages/esm-patient-chart-app/src/patient-flags/patient-flags-highlight-bar.component.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { Tag, Button, SkeletonPlaceholder } from '@carbon/react';
+import { ArrowRight, Close } from '@carbon/react/icons';
+import { useTranslation } from 'react-i18next';
+
+import { useCurrentPath, useFlagsFromPatient } from './hooks/usePatientFlags';
+import PatientFlags from './patient-flags.component';
+
+import styles from './patient-flags-highlight-bar.scss';
+
+interface PatientFlagsHighlightBarProps {
+  patientUuid: string;
+}
+
+const PatientFlagsHighlightBar: React.FC<PatientFlagsHighlightBarProps> = ({ patientUuid }) => {
+  const { t } = useTranslation();
+  const path = useCurrentPath();
+  const { flags, isLoadingFlags, flagLoadingError } = useFlagsFromPatient(patientUuid);
+  const filteredFlags = flags.filter((f) => !f.voided);
+
+  const [showHighlightBar, setShowHighlightBar] = useState(false);
+
+  const handleClick = () => {
+    setShowHighlightBar(true);
+  };
+
+  const handleClose = () => {
+    setShowHighlightBar(false);
+  };
+
+  if (path.includes('Summary')) {
```

Comment:
```suggestion
  if (path.includes('Patient Summary')) {
```

Reply:
Remember the path usually encodes the path so it can't be `Patient Summary`, I think looking for a path that includes `Summary` should suffice.

Reply:
But a decode can work so I am gonna use that.